Gems/ScriptCanvas/Code/Editor/View/Windows/Tools/UpgradeTool/UpgradeTool.cpp

```diff
@@ -92,7 +92,7 @@ namespace ScriptCanvasEditor
 
     void UpgradeTool::closeEvent(QCloseEvent* event)
     {
-        m_keepEditorAlive.reset();
+        // m_keepEditorAlive.reset();
```

Comment:
delete?

Reply:
all of upgrade tool .h/.cpp will leave us sooner rather than later, I'm not going to clean it up too much